src/ui_fsmenu/mapdetailsbox.cc

```diff
@@ -0,0 +1,178 @@
+#include "ui_fsmenu/mapdetailsbox.h"
+
+#include "graphic/image_cache.h"
+#include "map_io/map_loader.h"
+
+MapDetailsBox::MapDetailsBox(
+   Panel* parent, uint32_t, uint32_t standard_element_height, uint32_t padding, int32_t, int32_t)
```

Comment:
What are the unused parameters intended for?

Reply:
removed